tools/explorer/app.py

```diff
@@ -11,14 +11,133 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Main module for DBO explorer."""
+import ast
+import pyfiglet
 import sys
+from termcolor import colored
+from typing import List
 
-from lib import arg_parser
 from lib import explorer
 from lib import model
-import pyfiglet
+from lib.model import StandardField
+from lib import arg_parser
+
+def _InputFieldsManually(ontology) -> List[StandardField]:
+  """Method to take individual field inputs from user and return as a list.
+
+  Args:
+    ontology: An instance of OntologyWrapper class.
+
+  Returns:
+    A list of StandardField objects.
+  """
+  standard_field_list = []
+  field_number = input('Number of entity fields: ')
+  for i in range(1, int(field_number)+1):
+    field_name = input(f'{i}. What is the field name?\nname: ')
+    standard_field = model.StandardField(
+        standard_field_name=field_name,
+        namespace_name=''
+    )
+    if ontology.IsFieldValid(standard_field):
+      standard_field_list.append(standard_field)
+  return standard_field_list
+
+def _InputFieldsFromCSV(ontology) -> List[StandardField]:
+  """Method to take batch field inputs from user and return as a list
 
+  Args:
+    ontology: An instance of OntologyWrapper class.
+
+  Returns:
+    A list of StandardField objects
+  """
+  standard_field_list = []
+  raw_fields = input('Paste your fields here: ')
+  raw_field_list = raw_fields.replace(' ', '').split(',')
+  for field in raw_field_list:
+    standard_field = StandardField(
+        standard_field_name=field,
+        namespace_name=''
```

Comment:
these two methods can be refactored as they use the same code.

Reply:
The code is sightly different.